thirdeye/thirdeye-frontend/app/pods/components/modals/error-modal/component.js

```diff
@@ -0,0 +1,17 @@
+/**
```

Comment:
Add license header

Reply:
I added a ticket to the backlog for adding license headers to the frontend.